pkg/scheduler/framework/plugins/nodeports/node_ports_test.go

```diff
@@ -144,12 +146,18 @@ func TestNodePorts(t *testing.T) {
 			p, _ := New(nil, nil)
 			cycleState := framework.NewCycleState()
 			_, preFilterStatus := p.(framework.PreFilterPlugin).PreFilter(context.Background(), cycleState, test.pod)
+			if diff := cmp.Diff(test.wantPreFilterStatus, preFilterStatus); diff != "" {
+				t.Errorf("preFilter status does not match (-want,+got): %s", diff)
+			}
+			if preFilterStatus.IsSkip() {
+				return
+			}
 			if !preFilterStatus.IsSuccess() {
```

Comment:
I think we can remove this now.

Reply:
I think it's safer option to keep it for the future change even though the current PreFilter doesn't return error.

Reply:
> I think it's safer option to keep it for the future change even though the current PreFilter doesn't return error.

agree to sanposhiho.